File: trail1/모두 3의 배수/all-multiples-of-3.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        boolean flag = true;
        for (int i=0; i<5; i++) {
            int N = sc.nextInt();
            if (N % 3 != 0) {flag = false;}
        }
        int res = flag==true ? 1 : 0;
        System.out.println(res);
    }
}

File: trail1/모두 3의 배수/all-multiples-of-3.c
#include <stdio.h>
#include <stdbool.h>
int main() {
    bool flag = true;
    for (int i=0; i<5; i++) {
        int N;
        scanf("%d", &N);
        if (N % 3 != 0) flag = false;
    }
    int res = flag==true ? 1 : 0;
    printf("%d", res);
    return 0;
}